bindings/protocols/coap/index.html

```diff
@@ -1,181 +1,464 @@
+                    <td><code>"href": ..., "coap:method": ...</code></td>
+                </tr>
+                <tr>
+                    <td>Request to Forward Proxy</td>
+                    <td><code>"href": ..., "coap:method": ..., "coap:proxy": ...</code></td>
+                </tr>
+                <tr>
+                    <td>Caching: Validation</td>
+                    <td>n/a</td>
+                </tr>
+                <tr>
+                    <td>Conditional Requests</td>
+                    <td>n/a</td>
+                </tr>
+                <tr>
+                    <td>Content</td>
+                    <td><code>"coap:format": ...</code></td>
+                </tr>
+                <tr>
+                    <td>Content Negotiation</td>
+                    <td><code>"coap:accept": ...</code></td>
+                </tr>
+                <tr>
+                    <td>Observing Resources</td>
+                    <td><code>"subprotocol": "coap:observe"</code></td>
+                </tr>
+                <tr>
+                    <td>Block-Wise Transfer</td>
+                    <td><code>"coap:blockwise": { ... }</code></td>
+                </tr>
+                <tr>
+                    <td>Quick Block-Wise Transfer</td>
+                    <td><code>"coap:quickblockwise": { ... }</code></td>
+                </tr>
+                <tr>
+                    <td>Hop Limit</td>
+                    <td><code>"coap:hoplimit": ...</code></td>
+                </tr>
+                <tr>
+                    <td>OSCORE</td>
+                    <td><code>SecurityScheme</code></td>
+                </tr>
+                <tr>
+                    <td>EDHOC</td>
+                    <td><code>SecurityScheme</code></td>
+                </tr>
+            </tbody>
+        </table>
+    </section>
+    <section id="url">
+        <h2>URI Schemes</h2>
+        <p>
+            CoAP uses the following URI schemes for identifying CoAP resources
+            and providing a means of locating the resource:
+        </p>
+        <table class="def">
+            <thead>
+                <tr>
+                    <th>URI Scheme</th>
+                    <th>Reference</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>coap</td>
+                    <td>[[RFC7252]]</td>
+                </tr>
+                <tr>
+                    <td>coap+tcp</td>
+                    <td>[[RFC8323]]</td>
+                </tr>
+                <tr>
+                    <td>coap+ws</td>
+                    <td>[[RFC8323]]</td>
+                </tr>
+                <tr>
+                    <td>coaps</td>
+                    <td>[[RFC7252]]</td>
+                </tr>
+                <tr>
+                    <td>coaps+tcp</td>
+                    <td>[[RFC8323]]</td>
+                </tr>
+                <tr>
+                    <td>coaps+ws</td>
```

Comment:
for coap+ws as well

Should this instead be a subprotocol and the href should be ws ? This is a real question btw :)

Reply:
`coap+ws` and `coaps+ws` are actually official URI schemes, [specified in RFC 8323](https://datatracker.ietf.org/doc/html/rfc8323#section-8.3) :) So I think it is probably best to keep it in the href.